src/query.ts

```diff
@@ -25,28 +25,51 @@ export async function queryTask(taskid: string, resturl: string) {
         let aux = ZkWasmUtil.bytesToBN(task.aux);
         let fee = task.task_fee && ZkWasmUtil.convertAmount(task.task_fee); 
 
-        console.log("Task details: ");
-        console.log("    ", task);
-        console.log("    proof:");
-        aggregate_proof.map((proof: BN, index) => {
-            console.log("   0x", proof.toString("hex"));
-        });
-        console.log("    batch_instacne:");
-        batchInstances.map((ins: BN, index) => {
-            console.log("   0x", ins.toString("hex"));
-        });
-        console.log("    instacne:");
-        instances.map((ins: BN, index) => {
-            console.log("   0x", ins.toString("hex"));
-        });
-        console.log("    aux:");
-        aux.map((aux: BN, index) => {
-            console.log("   0x", aux.toString("hex"));
-        });
-        console.log("   fee:", fee);
+        if (enable_logs) {
+          console.log("Task details: ");
+          console.log("    ", task);
+          console.log("    proof:");
+          aggregate_proof.map((proof: BN, _) => {
+              console.log("   0x", proof.toString("hex"));
+          });
+          console.log("    batch_instacne:");
+          batchInstances.map((ins: BN, _) => {
+              console.log("   0x", ins.toString("hex"));
+          });
+          console.log("    instacne:");
+          instances.map((ins: BN, _) => {
+              console.log("   0x", ins.toString("hex"));
+          });
+          console.log("    aux:");
+          aux.map((aux: BN, _) => {
+              console.log("   0x", aux.toString("hex"));
+          });
+          console.log("   fee:", fee);
+        }
     }).catch((err) => {
         console.log("queryTask Error", err);
-    }).finally(() =>
+    }).finally(() => {
+      if (enable_logs) {
         console.log("Finish queryTask.")
+      }
+    });
+}
+
+export async function getAvailableImages(resturl: string, user_address : string) : Promise<Task[]> {
```

Comment:
This function can remain but still suggest allow tester to input the md5 of the image for doing pressure test